features/enlinkd/org.opennms.features.enlinkd.topogen/pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <parent>
+    <groupId>org.opennms.features</groupId>
+    <artifactId>org.opennms.features.enlinkd</artifactId>
+    <version>24.0.0-SNAPSHOT</version>
+  </parent>
+  <properties>
+    <bundle.symbolicName>org.opennms.features.enlinkd.topogen</bundle.symbolicName>
+    <bundle.namespace>org.opennms.features.enlinkd.topogen</bundle.namespace>
+  </properties>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.opennms.features.enlinkd</groupId>
+  <artifactId>org.opennms.features.enlinkd.topogen</artifactId>
+  <name>OpenNMS :: Features :: Enlinkd :: Topology Generator</name>
+  <packaging>bundle</packaging>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <instructions>
+            <Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
+            <Karaf-Commands>*</Karaf-Commands>
+          </instructions>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.opennms.maven.plugins</groupId>
+        <artifactId>features-maven-plugin</artifactId>
+        <configuration>
+          <bundles>
+            <bundle>mvn:org.opennms.features.enlinkd/org.opennms.features.enlinkd.topogen/${project.version}</bundle>
```

Comment:
@j-white what is the standing on this? Do we keep doing this or should we already start replacing this with manual `features.xml` files?

Reply:
Yeah, good point, let's avoid adding any new features via the `features-maven-plugin` and add them manually to the `features.xml` instead - will make migration easier.

Reply:
@mvrueden , @j-white uhm what do you want me to change here?

Reply:
Let's manually define the feature definition in `container/features/src/main/resources/features.xml` instead of using this plugin to generate it.

Reply:
done.

Reply:
hm but now it doesn't compile anymore... Anything else I need to change?